test/integration/deployers/container/containertests.go

```diff
@@ -0,0 +1,240 @@
+		utils.ExpectNoError(json.Unmarshal(configmapDataExport.Data.RawMessage, &configmapDataActual))
+		Expect(configmapDataExpected).To(Equal(configmapDataActual))
+
+		var componentData map[string]string
+		componentDataExport := &lsv1alpha1.DataObject{}
+		utils.ExpectNoError(state.Client.Get(ctx, types.NamespacedName{Name: "component", Namespace: state.Namespace}, componentDataExport))
+		utils.ExpectNoError(json.Unmarshal(componentDataExport.Data.RawMessage, &componentData))
+		Expect(componentData).To(HaveKey("name"))
+		Expect(componentData["name"]).To(Equal(inst.Spec.ComponentDescriptor.Reference.ComponentName))
+		Expect(componentData).To(HaveKey("version"))
+		Expect(componentData["version"]).To(Equal(inst.Spec.ComponentDescriptor.Reference.Version))
+
+		var (
+			contentData []map[string]interface{}
+		)
+		contentDataExport := &lsv1alpha1.DataObject{}
+		utils.ExpectNoError(state.Client.Get(ctx, types.NamespacedName{Name: "content", Namespace: state.Namespace}, contentDataExport))
+		utils.ExpectNoError(json.Unmarshal(contentDataExport.Data.RawMessage, &contentData))
+		Expect(contentData).To(HaveLen(4))
+
+		getFile := func(name string) map[string]interface{} {
+			for _, content := range contentData {
+				if content["name"] == name {
+					return content["stat"].(map[string]interface{})
+				}
+			}
+			return nil
+		}
+
+		verifyFile := func(file map[string]interface{}) {
+			Expect(file).ToNot(BeNil())
+			Expect(file["uid"]).To(BeEquivalentTo(1000))
+			Expect(file["gid"]).To(BeEquivalentTo(2000))
+			blueprintMode := file["mode"].(int64)
+			Expect((blueprintMode & unix.S_IWUSR) > 0).To(BeTrue())
+			Expect((blueprintMode & unix.S_IRUSR) > 0).To(BeTrue())
+		}
+
+		blueprint := getFile("blueprint.yaml")
+		verifyFile(blueprint)
+		deployExecution := getFile("deploy-execution.yaml")
+		verifyFile(deployExecution)
+		exportExecution := getFile("export-execution.yaml")
+		verifyFile(exportExecution)
+		script := getFile("script.py")
+		verifyFile(script)
+	})
+
+	It("should delete the configmap", func() {
+		createTargetAndDataObjects()
+
+		By("create installation")
+		inst := &lsv1alpha1.Installation{}
+		utils.ExpectNoError(utils.ReadResourceFromFile(inst, path.Join(testdataDir, "installation-1", "installation.yaml")))
+		inst.SetNamespace(state.Namespace)
+		utils.ExpectNoError(state.Create(ctx, inst))
+
+		By("Wait for installation to finish")
+		utils.ExpectNoError(lsutils.WaitForInstallationToFinish(ctx, f.Client, inst, lsv1alpha1.InstallationPhaseSucceeded, 5*time.Minute))
+
+		var configmapName string
+		utils.ExpectNoError(json.Unmarshal(doName.Data.RawMessage, &configmapName))
+
+		configmap := &v1.ConfigMap{}
+		utils.ExpectNoError(state.Client.Get(ctx, types.NamespacedName{Name: configmapName, Namespace: state.Namespace}, configmap))
+
+		utils.ExpectNoError(state.Client.Delete(ctx, inst))
+
+		err := wait.Poll(5*time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
```suggestion
		err := wait.Poll(1*time.Second, 5*time.Minute, func() (bool, error) {
```

just to make the tests faster?